static/scripts/app-login.js

```diff
@@ -2,17 +2,17 @@
 const formbtn = document.querySelector('.btn');
 const formInputs = document.querySelectorAll('.form-inputs');
 const formValues = document.querySelector('form');
+const inputsContainer = document.querySelector('.inputs-container');
 
 function keyDownEvents(){
-    formbtn.disabled = true;
+    formbtn.setAttribute('aria-disabled', 'true')
     for(let elem of formInputs){
         elem.addEventListener('input', () => {
             if(elem.value.length < 1){
-                formbtn.disabled = true
-                console.log(formbtn.disabled, elem.value, elem.value.length < 1)
+                formbtn.setAttribute('aria-disabled', 'true')
```

Comment:
python has [setattr method](https://docs.python.org/3/library/functions.html#setattr) and AFAIK it's good to use it when you DYNAMICALLY set the name of the attribute changed. It's not the case here, is it? Why are you using it? Is it some JS / ECMA convention?

Reply:
It's for [accessibility](https://css-tricks.com/making-disabled-buttons-more-inclusive/#aa-aria-to-the-rescue)! Coz we have ppl  who only use a keyboard (u can't access disabled button), blind ppl (there's a whole another topic about it)